src/common.js

```diff
@@ -248,6 +257,7 @@ function extend(target) {
 exports.extend = extend;
 
 // Common wrapper for all Unix-like commands
+// XXX: I'm unsure how this is going to integrate with plugins.
```

Comment:
So, we need to do some bikeshedding on how this is going to work.

Currently, `common.wrap` does mostly five things (all of which can be disabled with `notUnix: true`:

1. Parses `-options` if present.
2. Expands globs.
3. Expands `~`.
4. Converts ShellString to normal strings.
5. Handles the case where `common.error` (now `plugin.utils.error`) has to throw an error to stop execution, but we don't actually want to throw an error (because `config.fatal === false`).

Here's how I think this should be done:

1. Add a fourth parameter to `plugin.utils.plugin`, which is `options`. Then the first parameter is automatically `parseOptions`'d.
2. Add a fifth param, which is a list of options for `wrap`. (There's some slight tweaking I still want to do, primarily having `idx` default to 1, but that's not super relevant.)
3. All commands are wrapped by default.
4. `ShellString` conversions and `common.error` handling should be on by default, and should be a seperate option to disable.
5. Unless you say otherwise, your return value is `ShellString`'d